app/dfda/components/StudyCard.tsx

```
@@ -120,24 +191,26 @@ export default function StudyCard({ study }: StudyCardProps) {
-              />
-            )}
-            {study.studyCharts.pairsOverTimeLineChart && (
-              <div className="neobrutalist-container bg-gray-50 p-4">
-                <h3 className="neobrutalist-h3 mb-4">Data Over Time</h3>
-                <div dangerouslySetInnerHTML={{ 
-                  __html: study.studyCharts.pairsOverTimeLineChart.svg || '' 
-                }} />
-              </div>
-            )}
+            {Object.entries(study.studyCharts).map(([key, chart]) => (
+              chart && chart.highchartConfig ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)